test/factories/editions.rb

```diff
@@ -10,6 +10,12 @@
     previously_published { false }
     auth_bypass_id { SecureRandom.uuid }
 
+    after :create do |edition|
+      document = edition.document
+      document.update!(published_edition_id: edition.id) if edition.state.in?(Edition::PUBLICLY_VISIBLE_STATES)
+      document.update!(latest_edition_id: edition.id) if !edition.deleted? && (document.latest_edition_id.blank? || edition.id == document.editions.where(state: [Edition::PRE_PUBLICATION_STATES]).order(id: :desc).first&.id)
```

Comment:
There's some complexity here around making sure the correct values are set for this when objects are created in the factory. This is just me hacking around and should go into it's own service where it's unit tests

```
SetPublishedAndLatestEditionIDs.new(edition).call 
```

or something similar